docs/python/python-quick-start.md

```diff
@@ -0,0 +1,112 @@
+
+![Image of the Python UI highlights in VS Code.](/docs/python/images/quick-start/ui-tour.png)
+
+## Code Actions
+
+Code Actions (also known as Quick Fixes) are provided when there are Warnings or Errors to help fix issues you may be experiencing in your code. These helpful hints are displayed in the editor left margin as a lightbulb (💡). Click on the light bulb to display Code Action options which can come from extensions such as Python, Pylance, or VS Code itself. For more information about Code Actions, see [Python Quick Fixes](https://code.visualstudio.com/docs/python/editing#_quick-fixes).
+
+![Gif showing Code Actions in a Python project.](images/editing/quickFix.gif)
+
+## Python Commands
+
+Python commands can be accessed through the [Command Palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) (`kb(workbench.action.showCommands)`). From the Command Palette, you have access to various features from VS Code as well as installed extensions. Begin typing **“Python: “** to locate the commands accessible through the Python extension.
+
+![Gif demonstrating how to access Python commands in the Command Palette.](/docs/python/images/quick-start/cmd-plt-v2.gif)
+
+## Run, debug, and test
+
+Now that you are more familiar with Python in VS Code, let’s learn how to run, debug, and test your code.
+
+### Run
+
+There are a few ways to run Python code in VS Code.
+
+To run the Python script you have open on the editor, click the **Run Python File in Terminal** play button in the top-right of the editor.
+
+![Image showing the Run Python File in Terminal play button.](images/tutorial/run-python-file-in-terminal-button.png)
+
+There are also additional ways you can iteratively run snippets of your Python code within VS Code:
+-	Select one or more lines, then press `kbstyle(Shift+Enter)` or right-click and select **Run Selection/Line in Python Terminal**. This command is convenient for testing just a part of a file.
+-	From the Command Palette (`kb(workbench.action.showCommands)`), select the **Python: Start REPL** command to open a REPL terminal for the currently selected Python interpreter. In the REPL, you can then enter and run lines of code one at a time.
+
+### Debug
+
+The debugger is a helpful tool that allows you to inspect the flow of your code execution and more easily identify errors, as well as explore how your variables and data change as your program is run. You can start debugging by setting a breakpoint in your Python project by clicking in the gutter next to the line you wish to inspect.
+
+![Screenshot showing a debugger breakpoint in a Python program.](/docs/python/images/quick-start/breakpoint.png)
+
+To start debugging, initialize the debugger by pressing `F5`. Since this is your first time debugging this file, a configuration menu will open allowing you to select the type of application you want to debug. If it's a Python script, you can select "Python File".
```

Comment:
```suggestion
To start debugging, initialize the debugger by pressing `kbstyle(F5)`. Since this is your first time debugging this file, a configuration menu will open allowing you to select the type of application you want to debug. If it's a Python script, you can select **Python File**.
```